rocketchat/templates/mongodb-monitor.yaml

```diff
@@ -0,0 +1,21 @@
+{{ if and .Values.mongodb.metrics.enabled (not .Values.mongodb.metrics.serviceMonitor.enabled) (.Capabilities.APIVersions.Has "monitoring.coreos.com/v1") }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: rocketchat-mongodb
```

Comment:
Can we use one of the templated names here instead ? Will make it difficult to have multiple helmet deployments, names will conflict.